app/templates/buckets/details.html

```diff
@@ -39,7 +39,7 @@ <h2 class="heading-medium">Users with admin access</h2>
               </form>
 
               <!-- Button to revoke access -->
-              <form action="{{ url_for('users3buckets.delete', { id: users3bucket.id }) }}" method="post" class="inline-form clearfix">
+              <form action="{{ url_for('users3buckets.delete', { id: users3bucket.s3bucket.id }) }}" method="post" class="inline-form clearfix">
```

Comment:
See other comment: https://github.com/ministryofjustice/analytics-platform-control-panel-frontend/pull/124/files#r165109230

I think this was correct as you're revoking access, therefore deleting an `UserS3Bucket`, not an `S3Bucket`.